solidity/dashboard/Dockerfile

```diff
@@ -12,6 +16,7 @@ RUN apk add --update --no-cache \
 
 COPY package.json /app/package.json
 COPY package-lock.json /app/package-lock.json
+COPY .env* /app/
```

Comment:
Should we be copying .env? We set the relevant variables above, is that enough?

Reply:
I think we should. We set environments for 'machine' and inject into the dapp using .env file. check here https://create-react-app.dev/docs/adding-custom-environment-variables/#expanding-environment-variables-in-env